packages/component/src/Activity/private/FeedbackTextArea.tsx

```diff
@@ -0,0 +1,125 @@
+import { hooks } from 'botframework-webchat-api';
+import classNames from 'classnames';
+import React, {
+  forwardRef,
+  Fragment,
+  useCallback,
+  useRef,
+  type FormEventHandler,
+  type KeyboardEventHandler
+} from 'react';
+import useStyleSet from '../../hooks/useStyleSet';
+
+const { useUIState } = hooks;
+
+const TextArea = forwardRef<
```

Comment:
@OEvgeny or @lexi-taylor could you refactor this in another PR so we don't have two copies of same UI?